pkg/protos/bootstrap_grpc.pb.go

```diff
@@ -36,14 +38,19 @@ type AKSBootstrapTokenRequestClient interface {
 	// Step 2 of retrieving a bootstrap token; validates the attested data and the
 	// nonce, then generates and returns the bootstrap token to the client.
 	GetToken(ctx context.Context, in *TokenRequest, opts ...grpc.CallOption) (*TokenResponse, error)
+	AKSBootstrapTokenRequestSetConnection(conn *grpc.ClientConn)
```

Comment:
this shouldn't need to be part of the client interface, we only want to use it as an internal helper-type func

Reply:
ah okay sorry I was misreading this before, I thought this was going to be that setupClientConn func which I didn't want to expose

I think this is fine, I'd rather rename to something simpler like `SetConnection` or `SetGRPCConnection` or similar